corehq/apps/locations/templates/locations/manage/location_template.html

```diff
@@ -113,7 +113,7 @@
         <div class="modal-content">
           <div class="modal-header">
             <h4 class="modal-title">{% trans "Delete Location:" %} <span data-bind="text: name" /></h4>
-            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="{% trans_html_attr "Close" %}"></button>
```

Comment:
Probably work for a different PR, but I think we should work to remove `trans_html_attr` entirely.  Could you add a comment or docstring about it being deprecated?

Reply:
Oh, I see you mentioned that in your PR description already too.

Reply:
Yeah, I'm planning to do that. There are only ~200 usages of `trans_html_attr`, so that's not even the time it takes to sip a full cup of tea.